core/build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id 'java'
+}
+
+group 'org.mobilitydata'
+version "gtfs-validator-${System.getenv("versionTag")}"
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'com.univocity:univocity-parsers:2.9.0'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.10.0'
+    implementation 'com.google.guava:guava:29.0-jre'
+    implementation 'commons-validator:commons-validator:1.6'
+    testImplementation group: 'junit', name: 'junit', version: '4.13'
+    testImplementation "com.google.truth:truth:1.0.1"
```

Comment:
Google truth hey? :D

Reply:
I've seen Truth used in another project and it looks cool :)